.gitignore

```diff
@@ -0,0 +1,141 @@
+# generated C files
+crick/space_saving.c
+crick/space_saving_stubs.c
+crick/stats.c
+crick/tdigest.c
```

Comment:
Nit: Would something like `crick/*.c` work here?

Reply:
unfortunately not. The library contains some hand-coded C files, but these are generated by Cython when you build the project.